primitives/messages/src/target_chain.rs

```diff
@@ -171,14 +170,13 @@ const ALL_INBOUND_MESSAGES_REJECTED: &str =
 impl<MessagesProof: Parameter + Size, DispatchPayload> SourceHeaderChain
 	for ForbidInboundMessages<MessagesProof, DispatchPayload>
 {
-	type Error = &'static str;
 	type MessagesProof = MessagesProof;
 
 	fn verify_messages_proof(
 		_proof: Self::MessagesProof,
 		_messages_count: u32,
-	) -> Result<ProvedMessages<Message>, Self::Error> {
-		Err(ALL_INBOUND_MESSAGES_REJECTED)
+	) -> Result<ProvedMessages<Message>, VerificationError> {
```

Comment:
`/// Error message that is used in ForbidOutboundMessages implementation.`

The comment seems to be staled, should be `ForbidInboundMessages`

Reply:
You're right. Done.